packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -136,3 +143,50 @@ function validate(config, cwd, out, user_file) {
 		console.warn(`{\n  "extends": "${relative}"\n}`);
 	}
 }
+
+// <something><optional /*>
+const alias_regex = /^([^/]+)(\/\*)?$/;
```

Comment:
```suggestion
const alias_regex = /^(.+?)(\/\*)?$/;
```

I remember originally spending a good few minutes trying to get the non-greedy matching to work like in the `value_regex` before giving up to just `not slash`. Now that I think about it, I was looking at the wrong index of the match array. 😅